news/_posts/2020-06-12-product-features-methodology-at-prestashop.md

```diff
@@ -0,0 +1,109 @@
+---
+layout: post
+title: "Product development at PrestaShop"
+subtitle: "What happens in the team when working on new features."
+date: 2020-06-12 9:00:00
+authors: [SamuelPires]
+image: /assets/images/theme/meta-logo-build.png
+icon: icon-lab
+tags:
+- design
+- feature
+- process
+- product
+---
+
+In March 2020, the Core product team at PrestaShop, in charge of building and prioritizing new features, decided to work with a new tool: Productboard.
+
+You can learn more in this dedicated article about [how to use Productboard to share feedback on features the team is considering working on]().
+
+The team thought it was the right occasion to talk more about the product development at PrestaShop, the steps, the roles and responsibilities and what happens behind the scenes before a new feature is added to PrestaShop.
+
+
+## PrestaShop’s product development cycle
+
+### The Core product team
+At the Prestashop company, a team is dedicated to maintaining the PrestaShop open source project: the Core team.
+Its mission is to make sure the software is stable, usable, delivers value to its users and follows the industry standards.
+All this requires several people and several skills. Today the Core team is composed of developers, QA engineers, designers and product managers.
+Editing and maintaining a software like PrestaShop means fixing bugs but also adding improvements and new features.
+
+While bug fixing is most of the time handled directly by developers (from the Core team and the community 😉), features design and development are much more complex. Why? Because the goal of any feature is to help users achieve something, **behind every feature there should be a user need**.
+
+As a team working on a software, when you add a feature you are making the software more complex- more complex to maintain and sometimes more complex to use. So, on one hand, you can’t afford to add any useless feature and, on the other hand, you have to make sure what you add is usable.
+And that’s where product managers and designers are needed.
+Product managers are responsible for making sure the software delivers value to its users. And the first steps to do so are to know which features to improve or to create, and more importantly to know which users’ needs you have to solve.
+
+Once you know where to focus and what you are trying to solve, you have to design this feature, to make it tangible, usable and coherent with the existing software. That is the designers’ and the product content manager's responsibility.
```

Comment:
```suggestion
Once we know where to focus and what you are trying to solve, it is time to design the feature. It will allow us to create a tangible and usable solution, coherent with the software. That is the designers’ and the product content manager's responsibility.
```

Reply:
what we are trying to solve I guess. Or even better "what's the problem to solve"

Reply:
Updated ✅